src/new.rs

```diff
@@ -21,4 +23,6 @@ pub fn new(name: String) {
 
     let _ = Repository::init(format!("./{}", name))
         .map_err(|e| panic!("Failed to create an empty Git repository: {:?}", e));
+    
+    print("Your project `{}` has been created!\nNow, start your development environment by running `create-comit-app start-env`", name)
```

Comment:
Please use `println!`. Also, run `cargo fmt` to ensure the formatting is correct.